ReactiveCocoaTests/Swift/ObjectiveCBridgingSpec.swift

```diff
@@ -14,27 +14,97 @@ import ReactiveCocoa
 class ObjectiveCBridgingSpec: QuickSpec {
 	override func spec() {
 		describe("RACSignal.asSignalProducer") {
-			pending("should subscribe once per start()") {
+			it("should subscribe once per start()") {
+				var subscriptions = 0
+
+				let racSignal = RACSignal.createSignal { subscriber in
+					subscriber.sendNext(subscriptions++)
+					subscriber.sendCompleted()
+
+					return nil
+				}
+
+				let producer = racSignal.asSignalProducer() |> map { $0 as Int }
+
+				expect((producer |> single)?.value).to(equal(0))
+				expect((producer |> single)?.value).to(equal(1))
+				expect((producer |> single)?.value).to(equal(2))
 			}
 
-			pending("should automatically replace nil NSErrors") {
+			it("should automatically replace nil NSErrors") {
+				let racSignal = RACSignal.error(nil)
+				let producer = racSignal.asSignalProducer()
+				let result = producer |> last
+
+				expect(result?.error?.localizedDescription).to(equal("Nil RACSignal error"))
```

Comment:
I actually don't think we need this test—I don't know why I stubbed it out. Now that errors are parameterized, the type system proves that we _can't_ have a `nil` error, so it must be getting replaced by definition.

Yay types! :tada: